lib/foreman_maintain/cli/base.rb

```diff
@@ -60,6 +69,15 @@ def self.tags_option
       def self.interactive_option
         option ['-y', '--assumeyes'], :flag,
                'Automatically answer yes for all questions'
+
+        option ['-w', '--whitelist'], 'whitelist',
+               'Comma-separated list of labels of steps to be ignored' do |whitelist|
+          raise ArgumentError, 'value not specified' if whitelist.nil? || whitelist.empty?
```

Comment:
Clamp requires to raise ArgumentError here to print nice error to the user, therefore we need to use this here